_posts/2019-02-26-data-security-of-salesforce.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Data Security Of Salesforce"
```

Comment:
Of => of (There are some words that don't have to be capitalized in titles)
Also, I think "Data Security ```in``` Salesforce" sounds a bit better.